### PR TITLE
Modify API for population projections to accept new data

### PR DESCRIPTION
**Pull request by Catacola**
## Description of the change

I've added the real data to our export buckets and the shape of the data has changed slightly. This diff makes a few minor modifications so the FE can process the new shape.

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)

## Related issues

> Closes [#XXXX]

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Review by daschi — APPROVED** [review]
The code looks good but I think you need to rebase on master before merging! This looks like it was branched from `feature/vitals`

**Comment by Catacola** [comment]
> The code looks good but I think you need to rebase on master before merging! This looks like it was branched from `feature/vitals`

Thanks! Just fixed that.

File: src/core/PopulationTimeseriesChart/helpers.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const prepareData = (
 
   const projectedPopulation = historicalPopulation.slice(-1).concat(
     data
-      .filter((d) => d.simulationTag !== "HISTORICAL")
```
Comment (daschi) [inline src/core/PopulationTimeseriesChart/helpers.ts:80]: Do we also need to update HISTORICAL on lines 92, 95?

Also, do we need to update `POLICY_A` to be `PROJECTED`?

Comment (daschi) [reply src/core/PopulationTimeseriesChart/helpers.ts:80]: And also update the values in the demo data files

Comment (Catacola) [reply src/core/PopulationTimeseriesChart/helpers.ts:80]: My next PR will be to update the summary cards. Sadly, we need to shelve the admission and release projections until we can validate their output. I'll get that change in there.

**Pull request merged**